domain/removal/state/model/state_test.go

```diff
@@ -492,6 +492,14 @@ func (s *baseSuite) advanceModelLife(c *tc.C, modelUUID string, newLife life.Lif
 	c.Assert(err, tc.ErrorIsNil)
 }
 
+func (s *baseSuite) checkModelLife(c *tc.C, modelUUID string, expectedLife int) {
```

Comment:
It would be nicer is `expectedLife` were a `life.Life` here

Reply:
Yeah... I'll give it a crack.